frontend/Dockerfile

```
@@ -1,9 +1,22 @@
+
+WORKDIR /app
+
+RUN corepack enable
+
+COPY package.json pnpm-lock.yaml pnpm-workspace.yaml components.json ./
+COPY tsconfig.app.json tsconfig.json tsconfig.node.json vite.config.ts eslint.config.js index.html ./
+COPY public public
+COPY src src
+
+RUN pnpm install --frozen-lockfile
+RUN pnpm build
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)